libpod/runtime_ctr.go

```diff
@@ -797,7 +797,7 @@ func (r *Runtime) removeContainer(ctx context.Context, c *Container, force, remo
 	}
 
 	// Deallocate the container's lock
-	if err := c.lock.Free(); err != nil {
+	if err := c.lock.Free(); err != nil && !os.IsNotExist(err) {
```

Comment:
Can we still warn on `os.IsNotExist()`? If this happens in a case other than a reboot, it's an indication something could be seriously wrong.

Reply:
Do you mean Warning in log?

Reply:
I've pushed a new variant.